lib/taskana-core/src/main/java/pro/taskana/task/internal/builder/TaskBuilder.java

```diff
@@ -199,6 +199,11 @@ public TaskBuilder priority(Integer priority) {
     return this;
   }
 
+  public TaskBuilder manualPriority(Integer manualPriority) {
+    testTask.setManualPriority(manualPriority);
+    return this;
+  }
```

Comment:
I don't see a modification to `TaskBuilderTest#should_PopulateTask_When_UsingEveryBuilderFunction` for this :)